tests/components/synology_dsm/test_config_flow.py

```diff
@@ -4,7 +4,8 @@
 
 import pytest
 
-from homeassistant import data_entry_flow
+from homeassistant import config_entries, data_entry_flow, setup
```

Comment:
remove `config_entries` and add `SOURCE_SSDP` line 17

Reply:
remove `setup` and import only `from homeassistant.setup import async_setup_component` (seems like this on other tests)

Reply:
Should work as is

Reply:
Will switch as it will make it consistent with the rest of the test